custom_field/models.py

```diff
@@ -91,11 +88,16 @@ class CustomFieldValue(models.Model):
     A field instance -- contains the actual data.  There are many of these, for
     each value that corresponds to a CustomField for a given model.
     """
-    field = models.ForeignKey(CustomField, related_name='instance')
+
+    field = models.ForeignKey(
+        CustomField, related_name="instance", on_delete=models.CASCADE
+    )
     value = models.CharField(max_length=5000, blank=True, null=True)
     object_id = models.PositiveIntegerField(db_index=True)
-    content_type = models.ForeignKey(ContentType, blank=True, null=True)
-    content_object = fields.GenericForeignKey('content_type', 'object_id')
+    content_type = models.ForeignKey(
+        ContentType, blank=True, null=True, on_delete=models.CASCADE
```

Comment:
Looks like this is the only change? Everything else seems to be auto formatting 😄.